src/components/Main/Main.jsx

```diff
@@ -24,10 +25,12 @@ export default () => (
          <Route path="/forgotpassword" render={() => <ForgotPassword />} />
          <Route path="/newaccount" render={() => <NewAccountPage />} />
          <Route path="/editprofile" component={RequireAuth(EditProfilePage)} />
-         <Route path="/dashboard" component={RequireAuth(Dashboard)} />
+         {/* <Route path="/dashboard" component={RequireAuth(Dashboard)} /> */}
+         <Route path="/dashboard" render={() => <Dashboard />} />
```

Comment:
Change this back when you're done testing your code
```suggestion
         <Route path="/dashboard" component={RequireAuth(Dashboard)} />
```